server/methods.coffee

```diff
@@ -109,6 +154,9 @@ Meteor.methods
           # in the geoname lookup service's database.
           return
         geonamesById[geoname.id] = GeonameSchema.clean(geoname)
+        # Do not store alternate names because they aren't needed and take up
+        # a lot of space.
+        geoname.alternateNames = []
```

Comment:
This is a good idea. Should we remove the alternate names from existing incidents in a future chore?